specification/ai/ModelInference/tspconfig.yaml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  "service-dir":
+    default: "sdk/ai"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+  output-folder-base:
+    default: "data-plane/ModelInference/"
+emit:
+  - "@azure-tools/typespec-autorest"
+  - "@typespec/openapi3"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+  disable:
+    "@azure-tools/typespec-azure-core/casing-style": "Disabled since JSON payload in REST API does not conform to Azure guidelines with regards to casing"
+    "@azure-tools/typespec-azure-core/no-string-discriminator": "Use an extensible union instead of a plain string"
+    "@azure-tools/typespec-azure-core/bad-record-type": "We do want to use Record<unknown>, and not Record<string>. But this needs further investigation"
```

Comment:
Is these suppression valid? I didn't find a `Record<string>` in code.

Reply:
Here you have an example in the TypeSpec Azure Core repo: https://github.com/microsoft/typespec/blob/4d96eebab2381f7a90a279cf9d014689f7678fb1/packages/http-specs/specs/type/property/additional-properties/main.tsp#L26

Reply:
It's `Record<unknown>`: https://github.com/Azure/azure-rest-api-specs/blob/dfaac6f2bc8b60718186cbe87fd567626041c36b/specification/ai/ModelInference/models/chat_completions.tsp#L133

Reply:
Got it.

On the `Record<unknown>` and its spread in operation in routes.tsp, see this comment
https://github.com/Azure/azure-rest-api-specs/pull/28438#discussion_r1802429139

Reply:
@weidongxu-microsoft what's your improvement suggestion here?

Reply:
Make the op explicit use the `@body`
```ts
@route("chat/completions")
@route("chat/completions")
op getChatCompletions is Azure.Core.RpcOperation<
    { @body body: ChatCompletionsOptions} &
    AdditionalRequestHeaders
  ,
  ChatCompletions
>;
```

similar for `getEmbeddings` op

Reply:
Got you now! Apologies for the confusion! Fixed on https://github.com/Azure/azure-rest-api-specs/pull/32165/commits/547fb14bb3c5ab0c9171979e6fbcfe9acdd75a05

Reply:
@weidongxu-microsoft let me know what do you think!

Reply:
Thanks. This solves the spread problem.